### ISSUE TITLE
management command for resetting the jobma interview

### ISSUE THREAD

**Issue by pdpinch** [mitodl/bootcamp-ecommerce#1362]
When the jobma video service fails during an interview, we need to reset the applicant's state so they can submit again. 

After some trial and error, it appears that these are the necessary steps. 

1. Delete the existing interview record*
2. Set the bootcamp state to `AWAITING_USER_SUBMISSION` (it's usually in `AWAITING_REVIEW` when we learn the interview failed) 
3. run the task `refresh_pending_interview_links`  

### acceptance criteria

- [ ] * look into adding a `active` boolean to the interview record so that, instead of deleting the record, we can set it to be inactive. I'm not sure sure what additional code this might require, so please research and evaluate this change before implementing it
- [ ] management command that takes a user (email, username or id) and a bootcamp run (run code, or run id) and resets the jobma interview according to the steps above. 

### out of scope

I think a management command is appropriate for now. If the frequency of these errors continues to grow, we should consider adding a self-service UI for the application reviewer.

**Comment by mbertrand** [comment]
I agree that deleting the interview record is much simpler.  Not sure what the benefit would be of keeping old inactive interview records around, but if they are useful, then the above steps seem reasonable (or at least necessary) to implement that.

**Comment by pdpinch** [comment]
Thanks for researching this @arslanashraf7. After reading all the comments, I agree that we should go with deleting the record. 

I do like the idea of updating the audit table, if that's not too much trouble.
